fdbclient/NativeAPI.actor.cpp

```diff
@@ -883,8 +887,19 @@ void setNetworkOption(FDBNetworkOptions::Option option, Optional<StringRef> valu
 				throw invalid_option_value();
 			}
 			break;
-		case FDBNetworkOptions::KNOB: {
-			validateOptionValue(value, true);
+	    case FDBNetworkOptions::TRACE_FILE_IDENTIFIER:
```

Comment:
I guess this falls under the umbrella of the comment at the top of this switch (should these throw an error). We could move it to be the 5th option and change the option to say 5 to make that clearer.

Reply:
Done